tests/integration/test_writer.py

```diff
@@ -101,6 +101,7 @@ def test_write_cassandra(self):
             table='test_writer',
             consistency='ONE',
             mode='overwrite',
+            options={'confirm.truncate': True},
```

Comment:
let's comment why is this needed. Probably also add this to c* writer documentation - as it's also our public thing